Sources/Tokenizers/BPETokenizer.swift

```diff
@@ -38,13 +38,20 @@ class BPETokenizer: Tokenizer {
     private let addedTokens: Set<String>
     private let specialTokens: [String: Int]
     
+    public let unknownToken: String?
+    public let unknownTokenId: Int?
+    
     private let preTokenizer: PreTokenizer?
     private let normalizer: Normalizer?
     private let postProcessor: PostProcessor?
     private let decoder: Decoder?
     
     private let cleanUpTokenizationSpaces: Bool
 
+    class func unknownToken(from tokenizerConfig: Config) -> String? {
+        return tokenizerConfig.unkToken?.content?.stringValue ?? tokenizerConfig.unkToken?.stringValue
```

Comment:
The first part is the modern way; the second expression appears in distilgpt2, for example.